css/index.css

```diff
@@ -517,3 +519,110 @@ footer {
     padding: 40px 20px;
     text-align: center;
 }
+/* Toggle Button */
+    .toggle_btn {
+        display: none; 
+        flex-direction: column;
+        justify-content: space-around;
+        width: 30px;
+        height: 25px;
+        cursor: pointer;
+        margin-top: 6px;
+    }
+
+
+/* Desktop styles */
+@media (min-width: 1024px) {
+    .hidden {
+        display: block;
+    }
+}
+
+/* Show the toggle button on smaller screens */
+@media (max-width: 768px) {
+    .container {
+        padding-right: 15px;
+        padding-left: 15px;
+        margin-right: auto;
+        margin-left: auto;
+    }
+    .toggle_btn {
+        display: flex;
+        float: right;
+    }
+    #mobile_menu {
+        display: none; 
+        background-color: #fff;
+        position: absolute;
+        top: 60px; 
+        left: 0;
+        width: 100%;
+        box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+    }
+
+    .toggle_btn span {
+        width: 100%;
+        height: 3px;
+        background-color: #333;
+        border-radius: 5px;
+        transition: transform 0.3s ease, background-color 0.3s ease;
+    }
+    #mobile_menu ul {
+        list-style: none;
+        margin: 0;
+        padding: 0;
+    }
+    
+    #mobile_menu ul li {
+        text-align: center;
+        margin: 10px 0;
+    
+    }
+    
+    #mobile_menu ul li a {
+        text-decoration: none;
+        font-size: 18px;
+        color: #333;
+    }
+
+    #mobile_menu ul li {
+        display: block;
+    }
+    .hidden {
+        display: none;
+    }
+   
+    #About .image {
+        width: 100%;
+        float: left;
+    }
+    #Experience .col1 {
+        grid-template-columns: repeat(1, 1fr);
+        gap: 20px;
+    }
+    #Skills .Skills-1 {
+        flex: 0 0 auto;
+        width: 100%; 
+        float: left;
+    }
+    #Skills .Skills-2 {
+        flex: 0 0 auto;
+        width: 100%; 
+        float: left;
+        margin-left: 0;
+    }
```

Comment:
You could declare the properties just once, and then add just the difference to `.Skills-2`:

```css
    #Skills {
        flex: 0 0 auto;
        width: 100%; 
        float: left;

        .Skills-2 {
          margin-left: 0;
        }
    }
```